mobile-widgets/qml/Settings.qml

```diff
@@ -585,16 +585,16 @@ TemplatePage {
 						PrefDisplay.singleColumnPortrait = checked
 					}
 				}
-                TemplateLabel {
```

Comment:
What is going on with white space here (I cannot distinguish spaces from tabs here in github)?

Reply:
As stated in the pull request description, I have fixed incorrect indentation in this file as I was editing it anyway.